extras/jackson-support/src/main/java/com/palantir/remoting1/ext/jackson/ObjectMappers.java

```diff
@@ -34,6 +37,9 @@ public static ObjectMapper guavaJdk7Jdk8() {
                 .registerModule(new GuavaModule())
                 .registerModule(new ShimJdk7Module())
                 .registerModule(new Jdk8Module())
-                .registerModule(new AfterburnerModule());
+                .registerModule(new AfterburnerModule())
+                .registerModule(new JavaTimeModule())
+                .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
```

Comment:
Oh, I was hoping to get away with a token-test for OffsetDateTime and ZonedDateTime, but can add LocalDate too.